deps/build.jl

```diff
@@ -1,15 +1,37 @@
-#
-
-find_llvm() = begin
+#Returns the path as determined by llvm-config, or throws an error if LLVM
+#could not be found
+function find_llvm()
     if haskey(ENV, "TRAVIS")
         return readchomp(`which llvm-config-3.4`)
     end
     tmp = joinpath(JULIA_HOME, "llvm-config")
```

Comment:
Afk. Feel free to merge if/when it works.

On Sunday, August 14, 2016, Yichao Yu <notifications@github.com> wrote:

> In deps/build.jl
> <https://github.com/ihnorton/Clang.jl/pull/167#discussion_r74710607>:
>
> > @@ -1,15 +1,37 @@
> > -#
> > -
> > -find_llvm() = begin
> > +#Returns the path as determined by llvm-config, or throws an error if LLVM
> > +#could not be found
> > +function find_llvm()
> >      if haskey(ENV, "TRAVIS")
> >          return readchomp(`which llvm-config-3.4`)
> >      end
> >      tmp = joinpath(JULIA_HOME, "llvm-config")
>
> Cxx.jl only supports source build AFAIK. llvm-config doesn't seem to be
> included in current binary package.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ihnorton/Clang.jl/pull/167/files/8fa3cb85bf447edb58bf1110d45c0b588ffeda50#r74710607>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAUAGk-HX4GUSTVDxvsOTkbsPtV7Nv1mks5qf6GWgaJpZM4Jj8hQ>
> .
>